Partial migration of identity page (#3695)
w3f/polkadot-wiki: docs/learn/learn-identity.md

- Unknown: The default value, no judgement made yet.
- Reasonable: The data appears reasonable, but no in-depth checks (e.g. formal KYC process) were

>   performed (all the currently verified identities).

Reviewer: ```suggestion
  performed (all the currently verified identities on-chain).
```

> - Known Good: The registrar has certified that the information is correct (this step involves passport checks, and at the moment no account has known good identity, with the exception of registrars).

Reviewer: ```suggestion
- Known Good: The registrar has certified that the information is correct (this step involves verification of state issued identity documents, and at the moment no account has known good identity, with the exception of registrars).
```

:::

:::info Requesting judgement through Web3 Foundation Registrar

> If you requested judgement for your on-chain identity through the Web3 Foundation Registrar (i.e. Registrar #0) you will need to complete a few additional challenges. For more information visit [this support article](https://support.polkadot.network/support/solutions/articles/65000179747-how-to-use-the-w3f-registrar-page).

Reviewer: ```suggestion
If you requested judgement for your on-chain identity through the Web3 Foundation Registrar (i.e. Registrar #0) you will need to complete a few additional tasks. For more information visit [this support article](https://support.polkadot.network/support/solutions/articles/65000179747-how-to-use-the-w3f-registrar-page).
```
